addons/dev-helpers/setup-dev-env.sh

```diff
@@ -31,6 +31,10 @@ git clone https://github.com/inverse-inc/packetfence /usr/local/pf
 log_section "Set the safe.directory in git"
 git config --global --add safe.directory /usr/local/pf
 
+log_section "install epel and required header files"
+dnf install -y epel-release
```

Comment:
```suggestion
```
We do not need it.
libcurl and cjson are on our repo

Reply:
let me test it without installing epel release.